notebooks/official/automl/sdk_automl_tabular_regression_batch_bq.ipynb

```diff
@@ -277,7 +277,7 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    training_job = aiplatform.AutoMLTabularTrainingJob(
</pre><p>Something happened here. Let me remove the redundancy.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1265/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>